<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientmanagement</class>
 <widget class="QMainWindow" name="clientmanagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clientmanagement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true">#tab_3
{
    color: #b1b1b1;
    background-color: #323232;
}

#tab_3:item:hover
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #ca0619);
    color: #000000;
}

#tab_3:item:selected
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>681</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox
{
border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
      </property>
      <property name="title">
       <string> New Client</string>
      </property>
      <widget class="QLabel" name="id">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#id 
{
color: white;
}</string>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="fname">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#fname{
color:white;
}</string>
       </property>
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="lname">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lname{
color : white;
}</string>
       </property>
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="addresse">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#addresse{
color:white;

}</string>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
      <widget class="QLabel" name="email">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#email{
color: white;
}</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="phone">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>100</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#phone{
color:white;
}</string>
       </property>
       <property name="text">
        <string>Telephone</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_2
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_3
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_4
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_5
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_6
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}
</string>
       </property>
       <property name="text">
        <string> Add Client</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>691</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#tableView{
background-color : #FFFBFB
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>460</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#lineEdit_13
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>460</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*:hover
{
	color:#E9C9C9;
	border-radius:15px;
	background:#DF1010;
border: 2px solid #911111;
}</string>
      </property>
      <property name="text">
       <string> Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}
</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Search">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Sort">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}
</string>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchButton_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}
</string>
      </property>
      <property name="text">
       <string>SMS</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="Email">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>210</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#Email{
background-color:#363030;
color : white;
}</string>
      </property>
      <property name="text">
       <string>Send E-mail</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/mail.png</normaloff>../../../Downloads/mail.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>460</y>
        <width>23</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#refresh{
background-color :rgb(207, 110, 44);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/refresh.png</normaloff>../../../Downloads/refresh.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <property name="styleSheet">
      <string notr="true">#tab_4
{
    color: #b1b1b1;
    background-color: #323232;
}

#tab_4:item:hover
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #ca0619);
    color: #000000;
}

#tab_4:item:selected
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}</string>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>671</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_2
{
border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}
</string>
      </property>
      <property name="title">
       <string>Edit Info</string>
      </property>
      <widget class="QPushButton" name="UpdateButton">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}</string>
       </property>
       <property name="text">
        <string> Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="id2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#id2
{
color: white;
}</string>
       </property>
       <property name="text">
        <string> ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="fname2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#fname2
{
color:white;
}</string>
       </property>
       <property name="text">
        <string> First Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="lname2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lname2{
color:white;
}</string>
       </property>
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="addresse2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#addresse2{
color:white;
}</string>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
      <widget class="QLabel" name="email2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#email2{
color:white;
}</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="phone2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>110</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#phone2{
color:white;
}</string>
       </property>
       <property name="text">
        <string>Telephone</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_7
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_8
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_9
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_10
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_11
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_12
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0 #646464, stop: 1 #5d5d5d);
    padding: 1px;
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stats">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
color:white;
border-radius: 15px;
background:rgb(207, 110, 44);
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}</string>
      </property>
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="statistics">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
